Formula/racer.rb

```diff
@@ -0,0 +1,34 @@
+class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+
+  url "https://github.com/phildawes/racer/archive/v1.1.0.tar.gz"
+  sha256 "f969e66d5119f544347e9f9424e83d739eef0c75811fa1a5c77e58df621e066d"
+  head "https://github.com/phildawes/racer.git"
+
+  depends_on :rust => ["1.9.0", :build]
+
+  resource "rust_source" do
```

Comment:
There's been suggestions both ways on this point, I was told though that keeping the source around for a package wasn't allowed. So I did it here. Keeping it in rust is IMO a cleaner way to do this if it's allowed.